### DISCUSSION TITLE
Highlighting current row

### DISCUSSION THREAD

**Discussion by rkb97** [andmarti1424/sc-im#596] [Ideas]
Hello, does anybody know of a way to have the current selected row be highlighted like the current cell (but using a different color)? To make it easier to read row by row. All I've found in help doc was color regarding the current cell, current cell headings, and a way to alternate the column colors but not row colors.

**Comment by andmarti1424** [comment]
Not possible currently. See this: https://github.com/andmarti1424/sc-im/issues/20
